sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -30,18 +30,29 @@ Deserialization
 
 ## Using explicit cast
 
+When using protocol methods for advanced handling of RequestContext it is still possible to use the strongly typed models.
+There is an explicit cast operator that can be used to convert the protocol Response to the strongly typed model.
+There is also an explicit cast operator that can be used to convert the strongly typed model to the protocol RequestContent.
+
 Serialization
 
 ```C# Snippet:ExplicitCast_Serialize
-//TODO
+PetStoreClient client = new PetStoreClient(new Uri("http://somewhere.com"), new MockCredential());
+DogListProperty dog = new DogListProperty("myPet");
+Response response = client.CreatePet("myPet", (RequestContent)dog);
 ```
 
 Deserialization
 
 ```C# Snippet:ExplicitCast_Deserialize
-//TODO
+PetStoreClient client = new PetStoreClient(new Uri("http://somewhere.com"), new MockCredential());
+Response response = client.GetPet("myPet");
+DogListProperty dog = (DogListProperty)response;
+Console.WriteLine(dog.IsHungry);
 ```
 
+Given that explicit cast does not allow for serialization options we might also consider a static `FromResponse` and instance `ToRequestContent` methods.
```

Comment:
I think discussion of possible future design plans sounds strange in a sample. Should we just leave this out?

Reply:
We will clean this up for sure before merging into main.  Just putting a wiki together so all of us can see all the options and make a final decision on our direction.